README.md

```diff
@@ -35,84 +35,56 @@ import "github.com/swaggo/files" // swagger embed files
 ```
 
 ### Canonical example:
-
+Now assume you have implemented a simple api as following:
 ```go
-package main
-
-import (
-	"github.com/gin-gonic/gin"
-	swaggerFiles "github.com/swaggo/files"
-	ginSwagger "github.com/swaggo/gin-swagger"
-
-	_ "github.com/swaggo/gin-swagger/example/basic/docs" // docs is generated by Swag CLI, you have to import it.
-)
+//A get function which returns a Helloworld for you 
+func Helloworld(g *gin.Context)  {
+	g.JSON(http.StatusOK,"helloworld")
+}
 
-// @title Swagger Example API
-// @version 1.0
-// @description This is a sample server Petstore server.
-// @termsOfService http://swagger.io/terms/
+```
+So how to use gin-swagger on api above? Just follow the folling guide:)  
 
-// @contact.name API Support
-// @contact.url http://www.swagger.io/support
-// @contact.email support@swagger.io
+1. Add Comments for apis and main function with gin-swagger rules like following:  
+```go
+// PingExample godoc
+// @Summary ping example
+// @Schemes
+// @Description do ping
+// @Tags example
+// @Accept json
+// @Produce json
+// @Success 200 {string} Helloworld
+// @Router /example/helloworld [get]
+func Helloworld(g *gin.Context)  {
+	g.JSON(http.StatusOK,"helloworld")
+}
 
-// @license.name Apache 2.0
-// @license.url http://www.apache.org/licenses/LICENSE-2.0.html
 
-// @host petstore.swagger.io
-// @BasePath /v2
+//Comments used to generate swagger docs are igonred
+//We foucs on how to setup gin router in main
 func main() {
-	r := gin.New()
-
-	url := ginSwagger.URL("http://localhost:8080/swagger/doc.json") // The url pointing to API definition
-	r.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler, url))
-
-	r.Run()
+	r := gin.Default()
+
+    //Using this 
+	v1 := r.Group("/api/v1")
+	{
+		eg := v1.Group("/example")
+		{
+			eg.GET("/helloworld", Controller.Helloworld)
+		}
+	}
+	r.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler))
+	r.Run(":8080")
 }
 ```
 
-5. Run it, and browse to http://localhost:8080/swagger/index.html, you can see Swagger 2.0 Api documents.
-
-![swagger_index.html](https://user-images.githubusercontent.com/8943871/60704329-b7ab0680-9f36-11e9-9184-5c638c05e9c5.png)
-
-6. If you want to disable swagger when some environment variable is set, use `DisablingWrapHandler`
-
-### Example with disabling:
-
+2. Use `swag init` to generate a docs
+3. import the docs like this:
+Assume your project name is `MyGoSwagger`,don't forget the blank identifier!
```

Comment:
You should use the full import path `github.com/user/my-project` in examples in order to reduce confusion.